ScanCommand/src/main/java/io/ballerina/scan/ScanToolConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan;
+
+/**
+ * This class contains the constant variables used within Ballerina scan tool.
+ *
+ * @since 0.1.0
+ */
+public class ScanToolConstants {
```

Comment:
Why is this public?

Reply:
This class is created to hold all constant variables that are associated with the scan command and it follows the same approach as [constants](https://github.com/ballerina-platform/ballerina-lang/blob/48a9e694c2ef2e6528065f232036d4a3d191fe59/cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/Constants.java#L25)

Reply:
But there's no requirement for this to be public, is there? We can make it public if and when required.